docs/src/markdown/extensions/tasklist.md

```diff
@@ -85,7 +99,7 @@ Classes               | Description
     }
     ```
 
-??? settings "Custom Tasklist CSS"
```

Comment:
This is a details object, not a header.  Please change the syntax back to how it was.